css/README.md

```diff
@@ -410,3 +413,47 @@ When using HSL or RGB notation, always add a single space after a comma (,) and
 We prefer using mixins over functions. If a function is absolutely necessary, document where it is used and why it wasn't possible to create a mixin.
 
 **[⬆ back to top](#table-of-contents)**
+
+## CSS Modules
+
+[CSS Modules documentation](https://github.com/css-modules/css-modules)
+
+- Prefer using CSS Modules over global css code
+- Use camelCase for class names
```

Comment:
let's adjust the text about classnames above where it mentions where it's OK to use camelCase too.